manipulation/CMakeLists.txt

```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 2.8.3)
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
> Why did you bump this version without adding |add_compile_options|?
To be uniform across the repo? In this case +1.
However, maybe you missed add_compile_options?